### PR TITLE
Update components

### PR DESCRIPTION
**Pull request by fepitre**
QubesOS/qubes-issues#6982

### CODE REVIEW

**Comment by marmarek** [comment]
PipelineRetry

**Comment by marmarek** [comment]
@fepitre please preserve patches authors (based on git history, if no other info)

File: qemu/patches/series [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
-suppress_vmdesc.patch
-crypto-add-none-random-provider.patch
-qemu-register-ich6-ich9-soundhw.patch
-igd-fix-undefined-behaviour.patch
-igd-improve-legacy-vbios-handling.patch
-igd-move-enabling-opregion-access-to-libxl.patch
+0001-configure-add-enable-stubdom.patch
+0002-xen-handle-CONFIG_STUBDOM.patch
+0003-xen-hvm-handle-CONFIG_STUBDOM.patch
+0004-QubesGUI-display-option.patch
```
Comment (marmarek) [inline qemu/patches/series]: Typo:
```suggestion
0004-QubesGUI-display-options.patch
```

Comment (fepitre) [reply qemu/patches/series]: This is already corrected locally.

**Comment by fepitre** [comment]
> @fepitre please preserve patches authors (based on git history, if no other info)

As said few days ago, I'm currently waiting to have a working build first before adding all the details. Right now, I'm troubleshooting 6.1.0 which is also not working...

**Comment by fepitre** [comment]
PipelineRetry

**Comment by fepitre** [comment]
Good for openQA testing!

**Comment by fepitre** [comment]
PipelineRefresh

**Comment by marmarek** [comment]
PipelineRefresh

**Comment by marmarek** [comment]
>     * TC_41_HVMGrub_debian-11: [test_000_standalone_vm](https://openqa.qubes-os.org/tests/24717#step/TC_41_HVMGrub_debian-11/1)

```
[2021-11-12 14:44:50] qemu: -device VGA,vgamem_mb=16: failed to find romfile "vgabios-stdvga.bin"
```
So, it was necessary... The other one I guess too.

File: Makefile.stubdom [hunk 2]
```diff
@@ -280,18 +280,17 @@ build/qemu/config.status: build/qemu/.patched $(FULL_BUILDDEPS)
...
 		--extra-cflags="-DXEN_PT_LOGGING_ENABLED=1 ${QEMU_EXTRA_CFLAGS}" \
 		--extra-ldflags="${QEMU_EXTRA_LDFLAGS}" \
-		--prefix= 
+		--prefix=/usr
 
-build/qemu/i386-softmmu/qemu-system-i386: build/qemu/.patched build/qemu/config.status build/qemu/.gui-agent-copied-in
-	$(MAKE) -C build/qemu/roms seavgabios-{cirrus,stdvga}
-	$(MAKE) -C build/qemu/pc-bios/optionrom linuxboot.bin
+build/qemu/i386-softmmu/qemu-system-i386: build/qemu/.patched build/qemu/config.status
+	#$(MAKE) -C build/qemu/roms seavgabios-{cirrus,stdvga}
```
Comment (marmarek) [inline Makefile.stubdom]: This line should work as is (at least it works in qemu git checkout), then the resulting binary is copied to `pc-bios` dir.

**Comment by fepitre** [comment]
PipelineRefresh

**Comment by fepitre** [comment]
PipelineRetry

**Comment by marmarek** [comment]
Another one:
```
[2021-11-13 21:09:43] qemu: -device scsi-disk,drive=scsi0-hd3,scsi-id=3: 'scsi-disk' is not a valid device model name
```

Seems to require libxl changes - I'll try to find the right patch to backport.

**Review by marmarek — APPROVED** [review]

**Pull request merged**